components/General/index.tsx

```diff
@@ -135,26 +135,31 @@ export const Section: React.FC<SProps> = styled(({ className, children, label, t
 `;
 
 const clearLogos: Record<string, string> = {
+    TSLA: '/img/logos/currencies/tesla_clear.svg',
     ETH: '/img/logos/currencies/eth_clear.svg',
-    TEST1: '/img/logos/currencies/eth_clear.svg',
+    LINK: '/img/logos/currencies/link_clear.svg',
 };
 
 const logos: Record<string, string> = {
     TSLA: '/img/logos/currencies/tesla.svg',
     ETH: '/img/logos/currencies/eth.svg',
-    TEST1: '/img/logos/currencies/eth.svg',
     LINK: '/img/logos/currencies/link.svg',
-    DEFAULT: '/img/logos/currencies/tesla.svg',
 };
 
 interface LProps {
     className?: string;
-    ticker: string;
+    ticker: string | undefined;
     clear?: boolean; // true then display outlined image
 }
 
 export const Logo: React.FC<LProps> = styled(({ className, ticker, clear }: LProps) => {
-    return <img className={className} src={clear ? clearLogos[ticker] : logos[ticker] ?? logos['TSLA']} alt="logo" />;
+    return (
+        <>
+            {(ticker === 'TSLA' || 'ETH' || 'LINK') && ticker !== undefined ? (
```

Comment:
This boolean check is 
if (ticker === 'TSLA) or if ('ETH') or if ('LINK') && ticker !== undefined
my suggestion would be to create a ticker type in typescript and restrict ticker to ticker: TickerType | undefined
and then just have the ticker !== undefined

Reply:
done